.travis.yml

```diff
@@ -36,7 +36,9 @@ deploy:
 - provider: script
   script: ./scripts/release.sh
   skip_cleanup: true
-  if: branch IN (release, release-3.x)
+  on:
+    all_branches: true
+    condition: $TRAVIS_BRANCH =~ ^(release|release-3.x)$
```

Comment:
I might make this `^(release.*)$` so that if we rename `release` to `release-2.x` in the future (maybe after v3 final) we don't have to revisit

Reply:
Updated